tests/st/calicoctl/test_upgrade.py

```diff
@@ -451,29 +455,36 @@
              'selector': 'has(host-endpoint)'}
 }
 
+def strip_orch_label(data):
+    if 'metadata' in data:
+        if 'labels' in data['metadata']:
+            if 'projectcalico.org/orchestrator' in data['metadata']['labels']:
+                del(data['metadata']['labels']['projectcalico.org/orchestrator'])
```

Comment:
Why are we removing this label?  Is there any chance it might screw up someone who's using calico in a hybrid situation (i.e. with more than one orchestrator?)